FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -612,13 +609,16 @@ - (void)URLSession:(NSURLSession *)session
                     task:(NSURLSessionTask *)task
     didCompleteWithError:(NSError *)error {
   _isRequestInProgress = false;
+  [self->_settings updateRealtimeExponentialBackoffTime:self->_remainingRetryCount == gMaxRetries];
```

Comment:
not sure that we need to update backoff time here?

Reply:
This error occurs when the connection is interrupted client side do I think we should update the backoff time here too.